src/app/public/modules/country-field/country-field.component.scss

```diff
@@ -0,0 +1,81 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+@import "~intl-tel-input/build/css/intlTelInput";
+
+.sky-country-field-country-btn + ::ng-deep .sky-form-control {
+  border-left: none;
+}
+
+.sky-country-field-container {
+  width: 100%;
+
+  ::ng-deep .sky-dropdown-item .iti-flag {
```

Comment:
Would you be able to avoid using `::ng-deep` in favor of `:host-context`? We want to move away from ng-deep, as its being deprecated.

Reply:
Nope. We could use `host-context` if we were adding this file to the `dropdown` styles but you can't use `host-context` to get into a child's shadow dom. That works if you are wanting to apply a style based on something "above" the element.

Reply:
I also caution the idea that `ng-deep` is being deprecated. I understand that sentiment but it has been "deprecated" for over two years now and Angular has stated that they will not be removing it until an alternative is developed and that currently is not the case.

Reply:
Ok, thought I'd at least ask to see if there's another alternative. If not, I'm ok with what you've got.